app/grapefruit/base.toml

```diff
@@ -7,6 +7,10 @@ epoch = 0
 version = 0
 fwid = true
 
+[fmc]
+base-address = 0x60000000
+register-map = "../../drv/spartan7-loader/grapefruit/gfruit_top_map.json"
+
```

Comment:
It feels off that we have the base address here in the `base.toml` file instead of the `chip.toml` file.

Reply:
Good call, I've updated the PR to keep `fmc_nor_psram_bank_1` in `chip.toml` – with a comment telling you not to use it directly – then specifying that region by name in the manifest.